services/subscription.go

```diff
@@ -321,10 +320,12 @@ func (le RPCLogEvent) EthLogJSON() (models.JSON, error) {
 }
 
 func decodeABIToJSON(data hexutil.Bytes) (models.JSON, error) {
+	versionSize := 32
 	varLocationSize := 32
 	varLengthSize := 32
-	hex := []byte(string([]byte(data)[varLocationSize+varLengthSize:]))
-	return models.ParseJSON(bytes.TrimRight(hex, "\x00"))
+	prefix := versionSize + varLocationSize + varLengthSize
+	hex := []byte(string([]byte(data)[prefix:]))
```

Comment:
It'd be great to use a real ABI decoder function here instead of handling it manually, but that could definitely happen in another PR.